docs/cases/api/cases-api/cases-api-add-comment.asciidoc

```diff
@@ -11,8 +11,7 @@ NOTE: Console supports only Elasticsearch APIs. Console doesn't allow interactio
 
 ===== URL parts
 
-The URL must include the `case ID` of the case to which you are adding a
-comment. Call <<cases-api-find-cases>> to retrieve case IDs.
+The URL must include the `case ID` of the case that you are adding a comment to. Use <<cases-api-find-cases>> to retrieve case IDs.
```

Comment:
```suggestion
The URL must include the `case ID` of the case to which you are adding a comment. Use <<cases-api-find-cases>> to retrieve case IDs.
```

Reply:
I think both work! IMO, "to which" is a tad more formal (or at least reads/sounds a bit more formal) than my revision. I'm open to reverting my change if it's unnecessary though.

Reply:
This would be good for us to ask in the `#ux-writing` channel 👀 . For now, let's try to _not_ end sentences with a preposition if we can help it and we'll reconsider if other writers say otherwise.